data/config-schema/src/main/resources/net/corda/schema/configuration/crypto/1.0/corda.crypto.json

```diff
@@ -91,107 +92,46 @@
           },
           "additionalProperties": false
         },
-        "categories": {
-          "description": "Categories which the HSM can be used for",
+        "wrappingKeys": {
+          "description" : "Key derivation parameters for wrapping keys supplied in config",
```

Comment:
missing "default": {}, under wrappingKeys

Reply:
Why? At least one wrapping key must be specified